pages/index.vue

```diff
@@ -5,7 +5,7 @@
     >
         <masthead-secondary
             title="Modern Endangered Archives Program"
```

Comment:
pull title from craft too

Reply:
The Title will need to be update in craft then because it is currently not Modern Endangered Archives Program